src/WebJobs.Script.Grpc/Abstractions/IRpcServer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IRpcServer
+    {
+        void Start();
```

Comment:
Might make sense for the Start/Stop methods to both be async for symmetry?

Reply:
Yep, I have that change in a branch that's in progress

Reply:
I think I had the same feedback for @mamaso :)